cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -633,6 +635,10 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
                 result.templateInitContainerSecurityContext = template.getInitContainer().getSecurityContext();
             }
 
+            if (template.getClusterCaCert() != null && template.getClusterCaCert().getMetadata() != null) {
+                result.templateClusterCALabels = template.getClusterCaCert().getMetadata().getLabels();
+                result.templateClusterCAAnnotations = template.getClusterCaCert().getMetadata().getAnnotations();
```

Comment:
```suggestion
                result.templateClusterCaLabels = template.getClusterCaCert().getMetadata().getLabels();
                result.templateClusterCaAnnotations = template.getClusterCaCert().getMetadata().getAnnotations();
```
Just to be consistent with the other references to Ca